sql-cli/src/opensearch_sql_cli/__init__.py

```diff
@@ -3,4 +3,4 @@
 SPDX-License-Identifier: Apache-2.0
 """
 
-__version__ = "1.0.0"
+__version__ = "1.3.0.0"
```

Comment:
sql-cli versioning might be different from plugins. @zhongnansu what should this be? and are we releasing drivers with opensearch?

Reply:
I think SQL Drivers is having a release independent from OpenSearch, and is starting from 1.0.0 based on discussion I had with Zhongnan and Ani

Reply:
Should this be changed back to 1.0.0 ??

Reply:
yes, sql drivers/clients will be on a separate release schedule, you can revert this change